### ISSUE TITLE
Question on how to put "data" results to the DOM

### ISSUE THREAD

**Issue by phantompepper** [mscdex/ssh2#729]
Hi, I have a pretty simple idea that I've been racking my  head on - I am trying to get the "data" from the example on executing a shell command and putting that into an html element. It works great in the console, but I cant seem to see the obvious route. I am specifically testing this out in en Electron-Vue app and so far, all examples have worked great. I'm just trying to get the data to display in the DOM instead of the console. 

Here is the section from the example on this repo that I am referring to "data":
`conn.on("ready", function() {
          console.log("Client :: ready");
          
          conn.exec("ls", function(err, stream) {
            if (err) throw err;
            stream
              .on("close", function(code, signal) {
                console.log(
                  "Stream :: close :: code: " + code + ", signal: " + signal
                );
                conn.end();
              })
              .on("data", function(data) {
                
                console.log("STDOUT: " + data);
              })
              .stderr.on("data", function(data) {
                console.log("STDERR: " + data);
              })
          })
        })`

Specifically the STDOUT bit where I have succesfully executed 'ls' on my remote machine that I am connecting to. I just want to display the results of 'ls' on the webpage. I know my inexperience with Javascript is showing here - but I'd really appreciate any help with this!

**Comment by mscdex** [comment]
This kind of question is probably best suited for the [nodejs/help repo](https://github.com/nodejs/help/issues) or a place like StackOverflow.

**Comment by phantompepper** [comment]
Ok, thanks.

**Comment by phantompepper** [comment]
Ok - so I got this resolved and I'm finally seeing something in the DOM / UI. But now all I'm seeing are random numbers, while the console has no issue displaying the results of sending 'ls' to my remote computer. I've included a screenshot. Dev tools are open on the right side. Any ideas on why it does this?
![ls numbers](https://user-images.githubusercontent.com/5402776/44601607-3ae9c080-a79a-11e8-8fed-676c2e490624.PNG)
